frontend/packages/kubevirt-plugin/src/constants/status-constants.ts

```diff
@@ -3,6 +3,7 @@ export enum StatusSimpleLabel {
   Completed = 'Completed',
   Pending = 'Pending',
   Importing = 'Importing',
+  Copying = 'Copying disk image',
```

Comment:
please don't add it to StatusSimpleLabel since these should be shared acrros multiple enums. And `Copying disk image` is not generic enough. It should be included in `VMStatusSimpleLabel`.

Also please look at getSimpleLabel function. If you add a simple label, you should make sure it is working in places where this is used sorting/filtering (dashboard inventory and table filters)